src/test/functional/mutationServices.test/tagsMutationService-create.integration.ts

```diff
@@ -271,9 +281,9 @@ describe('Mutation for Tag: ', () => {
       variables,
     });
     expect(res.errors).to.undefined;
-    expect(res.data.createTags).to.deep.equal([
-      { name: 'kamille', savedItems: [{ id: '0' }] },
-    ]);
+    const data = res.data.createTags;
```

Comment:
same as above with the deep equal removal need to check length, or revert back to deep equal